fili-core/src/main/java/com/yahoo/bard/webservice/data/DruidHavingBuilder.java

```diff
@@ -70,16 +74,46 @@ public static Having buildMetricHaving(LogicalMetric metric, Set<ApiHaving> havi
      *
      * @param metric  The metric that the operation applied to.
      * @param having  The ApiHaving object
-     *
      * @return A single having representing the API Filter
      */
     public static Having buildHaving(LogicalMetric metric, ApiHaving having) {
         LOG.trace("Building having using metric: {} and API Having: {}", metric, having);
 
         HavingOperation operation = having.getOperation();
-        List<Having> havings = having.getValues().stream()
-                .map(value -> new NumericHaving(operation.getType(), metric.getName(), value))
-                .collect(Collectors.toList());
+        List<Double> values = having.getValues();
+        List<Having> havings = new ArrayList<>();
+        AndHaving andHaving = null;
+        if (operation.equals(HavingOperation.between) || operation.equals(HavingOperation.notBetween)) {
+            int size = 2;
```

Comment:
This as a local variable doesn't make much sense.  Either just use '2' or make it a constant at the class level.